test/useLocalStore.test.tsx

```diff
@@ -455,3 +455,69 @@ describe("is used to keep observable within component body", () => {
         restore()
     })
 })
+
+describe("enforcing actions", () => {
+    it("'never' should work", () => {
+        mobx.configure({ enforceActions: 'never' })
+        const restore = mockConsole() // to ignore React showing caught errors
+        const { result } = renderHook(() => {
+            useLocalStore(
+                props => ({
+                    count: 10,
+                    inc() {
+                        this.count += 1
+                    },
+                    get mul() {
+                        return this.count * props.multiplier
+                    }
+                }),
+                { multiplier: 2 }
+            )
+        })
+        expect(result.error).toBeUndefined();
+        restore()
+        mobx.configure({})
```

Comment:
I think this isn't doing anything actually since no value for `enforceActions` is being passed.

https://github.com/mobxjs/mobx/blob/a5cb65926621266716f8d11d716bf337e60dc3c1/src/api/configure.ts#L25

Probably the best to use `afterEach` and reset it specifically to `never` as default.

Reply:
Good call. I should have looked at the source a bit more closely :)